src/components/DocumentSelector/index.js

```diff
@@ -30,32 +33,104 @@ class DocumentSelector extends Component {
 
   handleSelect = (e, value) => {
     e.stopPropagation()
-
     const { setDocumentType, nextStep } = this.props
     setDocumentType(value)
     nextStep()
   }
 
-  renderOption = (option) => {
-    return (
-      <div
-        class={style.option}
-        onClick={(e) => this.handleSelect(e, option.value)}
-      >
-        <div class={`${style.icon} ${style[option.icon]}`}></div>
-        <span>{option.label}</span>
+  renderOption = (option) => (
+    <div
+      class={style.option}
+      onClick={e => this.handleSelect(e, option.value)}
+    >
+      <div className={`${style.icon} ${style[option.icon]}`}></div>{/*`*/}
+      <div className={style.content}>
+        <div className={style.optionMain}>
+          <p className={style.label}>{option.label}</p>
+          {option.hint &&
+            <div className={style.hint}>{option.hint}</div>
+          }
+          {option.warning &&
+            <div className={style.warning}>{option.warning}</div>
+          }
+        </div>
+        {option.eStatementAccepted &&
+          <div className={style.tag}>{
+            this.props.i18n.t('document_selector.proof_of_address.estatements_accepted')
+          }</div>
+        }
       </div>
-    )
-  }
+    </div>
+  )
 
   render() {
     const documentOptions = this.getOptions()
+    const { className } = this.props
     return (
-      <div class={style.selector}>
-        {documentOptions.map((op) => this.renderOption(op))}
+      <div className={classNames(style.wrapper, className)}>
+        {documentOptions.map(this.renderOption)}
       </div>
     )
   }
 }
 
-export default DocumentSelector;
+const snakeCase = str => str.replace(/_/g, '-')
```

Comment:
minor: snakeToDashCase ?

Reply:
@rfreitas, I named it `snakeCase`, but really I wanted to name it `kebabCase`. What do you think about `snakeToKebabCase`?

Reply:
I think snakeToKebab is probably the best function name I've come across.